aardwolf-models/Cargo.toml

```diff
@@ -36,4 +37,4 @@
 
 [dev-dependencies]
 env_logger = "0.10"
-serde_derive = "1.0"
+serde_derive = "1.0"
```

Comment:
**Suggestion:** The `serde_derive` package is added as a dev-dependency, but it's not clear if it's used in the tests or benchmarks. If it's not used, consider removing it to reduce the project's dependencies.
```suggestion

```

Reply:
I feel like serde_derive is required elsewhere...